### DISCUSSION TITLE
Docker - v local content does not correspond to files inside Docker after mounting

### DISCUSSION THREAD

**Discussion by 955s** [dani-garcia/vaultwarden#4310] [Q&A]
The/vw data/directory was not created locally after running the script
I will first restore the backup file/vw data/and then dock v to mount it. I found that the mounted content does not correspond to the local file
![image](https://github.com/dani-garcia/vaultwarden/assets/14900130/4699dfdd-2942-4736-aaad-29f5fada6f78)

**Comment by pedrojreis** [comment]
Hi @955s 

Can you try and do a restore without the `db.sqlite3-shm` file and see if that works?
If that doesn't work, just restore the `db.sqlite3` and `db.sqlite3-wal` only and see what's the outcome.

What I Imagine you are getting it's some error when trying to recover the DB and getting a cleaning state instead of a non functioning VW.

**Comment by 955s** [comment]
It has been resolved. It is a problem with installing Docker with Snap install Docker. Installing Docker with Snap will cause Docker - v to not mount properly. Installing Docker with apt. install Docker.io will not have this problem. Thank you







                       
原始邮件
                       
                     

发件人："Pedro Reis"< ***@***.*** &gt;;

发件时间：2024/2/3 20:53

收件人："dani-garcia/vaultwarden"< ***@***.*** &gt;;

抄送人："955s"< ***@***.*** &gt;;"Mention"< ***@***.*** &gt;;

主题：Re: [dani-garcia/vaultwarden] Docker - v local content does notcorrespond to files inside Docker after mounting (Discussion #4310)




 
Hi @955s
 
Can you try and do a restore without the db.sqlite3-shm file and see if that works?
 If that doesn't work, just restore the db.sqlite3 and db.sqlite3-wal only and see what's the outcome.
 
What I Imagine you are getting it's some error when trying to recover the DB and getting a cleaning state instead of a non functioning VW.
